src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -68,10 +125,13 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
     static loco_global<uint16_t, 0x0052334A> _52334A;
     static loco_global<uint16_t, 0x0052334C> _52334C;
 
+    // _tabObjectCounts can be integrated after implementing sub_473A95
     static loco_global<uint16_t[33], 0x00112C181> _tabObjectCounts;
-    static loco_global<tabPosition[36], 0x0112C21C> _tabInformation;
+
+    static TabPosition _tabInformation[36]; // was 0x0112C21C
```

Comment:
This is really a fixed sized vector @ZehMatt we could do with adding an implementation of a fixed sized vector to core. We currently have locofixedvector but thats a wrapper for internal structures we can't modify atm.